MASH-FRET/source/simulations/arb2phtn.m

```diff
@@ -1,25 +1,11 @@
-function dat = arb2phtn(varargin)
```

Comment:
First, a comment to your commit description:
The photon to image counts conversion does use default values for offset=0, K = 57 and eta = 0.95. 
Second, your camera noise model parameters are very good.
Third, there might be a problem with the noise added to the offset. 
Fourth, offset = dark image counts.

Reply:
Skype ^^